permissions/plugin-support-core.yml

```diff
@@ -14,6 +14,7 @@ developers:
 - "dnusbaum"
 - "egutierrez"
 - "mramonleon"
+- "allan_burdajewicz"
```

Comment:
Is it the GitHub id here ? thus @Dohbedoh and not allan_burdajewicz ??

Reply:
It's not. See readme.

Reply:
Effectively

> user accounts in LDAP

(That's why I have the same account everywhere :-) )